.travis.yml

```diff
@@ -9,6 +9,9 @@ matrix:
   allow_failures:
     - rust: nightly
   fast_finish: true
+before_script:
+  - rustup component add rustfmt-preview
+  - which rustfmt
```

Comment:
What does this do?

Reply:
rustfmt used for tests, and rustfmt is not installed by default, so these lines installed it.